### ISSUE TITLE
Should an unpaired quote throw an error?

### ISSUE THREAD

**Issue by gaborcsardi** [tidyverse/glue#219]
Probably. Cf. https://github.com/r-lib/cli/issues/333

```r
❯ str(glue::glue("{this doesn't work}"))
 'glue' chr ""

❯ str(glue::glue("{this does not work}"))
Error in parse(text = text, keep.source = FALSE) :
  <text>:1:6: unexpected symbol
1: this does
```

**Comment by gaborcsardi** [comment]
@jimhester would you review a PR for this?

**Comment by jimhester** [comment]
yeah

**Comment by gaborcsardi** [comment]
This is now fixed in #226 I believe..
